analyzers/tests/SonarAnalyzer.UnitTest/Rules/ObsoleteAttributesTest.cs

```diff
@@ -24,25 +24,49 @@
 namespace SonarAnalyzer.UnitTest.Rules;
 
 [TestClass]
-public class RemoveObsoleteCodeTest
+public class ObsoleteAttributesTest
 {
-    private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.RemoveObsoleteCode>();
-    private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.RemoveObsoleteCode>();
+    private readonly VerifierBuilder builderExplanationNeededCS =
+        new VerifierBuilder<CS.ObsoleteAttributes>().WithOnlyDiagnostics(CS.ObsoleteAttributes.S1123);
+    private readonly VerifierBuilder builderExplanationNeededVB =
+        new VerifierBuilder<VB.ObsoleteAttributes>().WithOnlyDiagnostics(CS.ObsoleteAttributes.S1123);
+    private readonly VerifierBuilder builderRemoveCS =
+        new VerifierBuilder<CS.ObsoleteAttributes>().WithOnlyDiagnostics(CS.ObsoleteAttributes.S1133);
+    private readonly VerifierBuilder builderRemoveVB =
+        new VerifierBuilder<VB.ObsoleteAttributes>().WithOnlyDiagnostics(CS.ObsoleteAttributes.S1133);
+
+    [TestMethod]
+    public void ObsoleteAttributesNeedExplanation_CS() =>
```

Comment:
```suggestion
    public void ObsoleteNeedExplanation_CS() =>
```

Reply:
Der Name war schon so und ist deckungsgleich mit dem Namen der Test-Datei ("ObsoleteAttributesNeedExplanation.cs"). Soll ich dann auch die Dateien umbenennen?

Reply:
Ah. I couldn't see that while reviewing on the phone. If it wasn't changed you should keep it. There is no need to make such a minor change.